DSCResources/MSFT_xKDSKey/MSFT_xKDSKey.psm1

```diff
@@ -0,0 +1,80 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory)]
+        [DateTime] $StartTime
+    )
+
+    try
+    {
+        $KDSKeys = @(Get-KDSRootKey)
+        return @{
+            AttributeOfWrongFormat = $KDSKeys[0].AttributeOfWrongFormat
+            KeyValue = $KDSKeys[0].KeyValue
```

Comment:
why is get returning property values not defined in the schema ?  #needsdesignreview